packages/bpk-component-banner-alert/src/BpkBannerAlert.js

```diff
@@ -45,14 +46,16 @@ export const ALERT_TYPES = {
   NEUTRAL: 'neutral',
 };
 
+type AlertTypeValue = $Values<typeof ALERT_TYPES>;
```

Comment:
Should we export this and similar types? It makes it easier to reuse them in the docs for example.